### PR TITLE
SVGLoader: Implement units

### LINKED ISSUE #17630: SVGLoader - issue with unit conversion
##### Description of the problem
[edited issue]
I did some tests with loading an SVG drawing in three.js and I found something odd about the scaling/units. 

I export a drawing from my CAD (units: meter) and apply a scaling (say 1:500). My CAD app uses the pt unit to set the size of the SVG geometry. When loaded by the three.js SVGLoader, the SVG units are applied in three.js (1 SVG unit (in my case pt) = 1 three.js unit). I assume my scene to be in meter, so in that case, I know which factor to use for scaling in three.js ((1/export scale) * 0.352778))

##### Three.js version

- [x] Dev
- [ ] r109
- [ ] ...
(probably also in older versions)

##### Browser

- [ ] All of them
- [x] Chrome
- [ ] Firefox
- [ ] Internet Explorer

##### OS

- [ ] All of them
- [x] Windows
- [ ] macOS
- [ ] Linux
- [ ] Android
- [ ] iOS

### PR DESCRIPTION
**Pull request by yomboprime**
Fixes #17630

Implemented the 6 most common units: ```px```, ```mm```, ```in```, ```cm```, ```pt```, ```pc```.

The ```em```, ```ex``` and ```%``` unit specifiers were not possible to implement because there is no text implemented to refer to, nor viewports.

Default DPI are 90, which can be changed in ```svgLoader.defaultDPI```.

Also the ```svgLoader.defaultUnit``` member can be set to one of the 6 unit strings, to set the loader output units (default is ```px```, i.e. user units)

For testing, I added a new SVG  test file (named "Units") in which three rectangles with equal widths are defined in different units.

### CODE REVIEW

**Comment by Mugen87** [comment]
For testing: http://raw.githack.com/yomboprime/three.js/svgunits/examples/webgl_loader_svg.html

**Review by Mugen87 — APPROVED** [review]

File: examples/js/loaders/SVGLoader.js [hunk 1]
```diff
@@ -8,6 +8,12 @@ THREE.SVGLoader = function ( manager ) {
 
 	THREE.Loader.call( this, manager );
 
+	// Default dots per inch
+	this.defaultDPI = 90;
+
+	// Accepted units: 'mm', 'cm', 'in', 'pt', 'pc', 'px'
+	this.defaultUnit = "px";
```
Comment (Mugen87) [inline examples/js/loaders/SVGLoader.js]: You might want to add these two new properties to the TS file.

**Comment by yomboprime** [comment]
Done 👍

**Comment by mrdoob** [comment]
Thanks!

**Pull request merged**